### DISCUSSION TITLE
Header ändern, Vorlage nutzen

### DISCUSSION THREAD

**Discussion by jsk12287** [codeispoetry/sunflower#512] [Fragen und Antworten]
Ich habe gesehen, dass es Vorlagen gibt zu anderen Headern als in der Standardeinstellung. Auch der Bundesverband und auch der KV Berlin Mitte nutzen diese. Wie kann ich den Header meiner Webseite ändern?

**Comment by FloJuh** [comment]
Unter _Sunflower < Einstellungen_ Layout für den Header
